llpc/CMakeLists.txt

```diff
@@ -144,16 +144,21 @@ set_compiler_options(llpc ${LLPC_ENABLE_WERROR})
 
 ### Defines/Includes/Sources ###########################################################################################
 if(ICD_BUILD_LLPC)
-    set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${XGL_LLVM_BUILD_PATH}/lib/cmake/llvm)
+    list(APPEND CMAKE_MODULE_PATH
+        ${XGL_LLVM_BUILD_PATH}/lib/cmake/llvm
+        ${XGL_LLVM_BUILD_PATH}/${CMAKE_CFG_INTDIR}/lib/cmake/llvm   # Workaround for multi-config generators (e.g. VS)
+    )
     include(LLVMConfig)
     message(STATUS "LLVM executables: " ${LLVM_TOOLS_BINARY_DIR})
     message(STATUS "LLVM libraries: " ${LLVM_BUILD_LIBRARY_DIR})
-    execute_process(
-        COMMAND ${LLVM_TOOLS_BINARY_DIR}/llvm-config --libs amdgpu analysis bitreader bitwriter codegen irreader linker mc passes support target transformutils
-        OUTPUT_VARIABLE LLVM_LINK_FLAGS
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-    )
-    message(STATUS "LLVM link options:" ${LLVM_LINK_FLAGS})
+    if(EXISTS ${LLVM_TOOLS_BINARY_DIR}/llvm-config)
+        execute_process(
+            COMMAND ${LLVM_TOOLS_BINARY_DIR}/llvm-config --libs amdgpu analysis bitreader bitwriter codegen irreader linker mc passes support target transformutils
+            OUTPUT_VARIABLE LLVM_LINK_FLAGS
+            OUTPUT_STRIP_TRAILING_WHITESPACE
+        )
```

Comment:
Do we need to run llvm-config at all (I though it was deprecated)?
I don’t like putting it behind an exists flag. If it’s not needed we should remove it. If it’s needed, it should fail here instead of miscompile or fail later due to variables missing.